scene/resources/packed_scene.cpp

```diff
@@ -380,7 +380,27 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 				//if node was not part of instance, must set its name, parenthood and ownership
 				if (i > 0) {
 					if (parent) {
-						parent->_add_child_nocheck(node, snames[n.name]);
+						Node *existing = nullptr;
+						if (Engine::get_singleton()->is_editor_hint()) {
+							existing = parent->get_node_or_null((String)snames[n.name]);
+						}
+						if (existing) {
+							// There's already a node in the same parent with the same name.
+							// This means that somehow the node was added both to the scene being
+							// loaded and another one instantiated in the former, maybe because of
+							// manual edition, or a bug in scene saving, or a loophole in the workflow
+							// (with any of the bugs possibly already fixed).
+							// Bring consistency back by letting it be assigned a non-clashing name,
+							// regardless the (broken) intent was not to have a new node, but merely
```

Comment:
this sentence also seems kinda broken after the regardless

Reply:
I've removed that sentence, which is not especially important. But, for the records, this is a rewording of what I meant:
```
// An alternative would be to try to recover the likely original
// intention that the extra node is a reference to another one in an
// instance, and thus is there just to override property values.
```